curvefs/src/client/client_operator.cpp

```diff
@@ -35,15 +37,18 @@ using ::curvefs::mds::topology::PartitionTxId;
                << ", DebugString = " << DebugString();
 
 RenameOperator::RenameOperator(uint32_t fsId,
+                               const std::string& fsname,
```

Comment:
> Already have fsId, why need fsname?

we need to lock the namespace of `fsName`, if we only pass fsId to `MDS`, we should ne more operation to get `fsName` by `fsId`.

> fsname to fsName

fixed

Reply:
The fsId is also unique. Can fsId be locked replace fsName？

Reply:
> The fsId is also unique. Can fsId be locked replace fsName？

not yet~ because other operator lock FS by fsname in `FsManager`:

```
FSStatusCode FsManager::DeleteFs(const std::string& fsName) {
    NameLockGuard lock(nameLock_, fsName);
   ...
}
```